tests/tensor/test_nlinalg.py

```diff
@@ -609,8 +609,10 @@ def test_kron_commutes_with_inv(self):
         for shp0, shp1 in zip(
             [(2, 3), (2, 3), (2, 4, 3)], [(6, 7), (4, 3, 5), (4, 3, 5)]
         ):
-            if len(shp0) == 3 or len(shp1) == 3:
-                continue
+            if (pytensor.config.floatX == "float32") & (
+                len(shp0) == 3 or len(shp1) == 3
```

Comment:
This skips test 2 and 3, but we only need to skip test 3. I'd just add an enumerate to the loop, and check if `i == 2`

Reply:
Alternatively, change the `or` to `and`